components/utils.py

```diff
@@ -1587,143 +1332,22 @@ def wait_for_workflow(deployment_id,
     return False
 
 
-def _wait_for_execution(execution_id, headers):
-    poll_interval = 2
-    while True:
-        res = _list_executions_with_retries(headers, execution_id)
-        content = json.loads(res.readlines()[0])
-        execution_item = content['items'][0]
-        execution_status = execution_item['status']
-        if execution_status == 'terminated':
-            return True
-        elif execution_status == 'failed':
-            ctx.abort_operation('Execution with id {0} failed'.
-                                format(execution_id))
-        sleep(poll_interval)
-
-
-def _list_executions_with_retries(headers, execution_id, retries=6):
-    count = 0
-    err = 'Failed listing existing executions.'
-    url = 'http://localhost/api/{0}/executions?' \
-          '_include_system_workflows=true&id={1}'.format(REST_VERSION,
-                                                         execution_id)
-    while count != retries:
-        res = http_request(url, method='GET', headers=headers)
-        if res.code != 200:
-            err = 'Failed listing existing executions. Message: {0}' \
-                .format(res.readlines())
-            ctx.logger.error(err)
-            sleep(2)
-        else:
-            return res
-    ctx.abort_operation(err)
-
-
-def create_maintenance_headers(upgrade_props=True):
+def create_maintenance_headers():
```

Comment:
is this actually still needed? this is only used for waiting for sanity execution status

Reply:
Wil check